hexvexed-0.0.1.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="Hexagon game based  for haiku"
+DESCRIPTION="This is a hexagon based tile puzzle game forked from DarkWyrm's BeVexed \
+code to support 6 sides instead of four."
+
+HOMEPAGE="https://bitbucket.org/scottmc/hexvexed"
+COPYRIGHT="2009-2016 Scott McCreary"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://bitbucket.org/scottmc/hexvexed/get/68437a22c788.zip"
+SOURCE_DIR="scottmc-hexvexed-68437a22c788"
+CHECKSUM_SHA256="35e0904d91f15fefe801cdb7b9f9bdf4dc925de69f5b8818bec24cc4c22dc6d7"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+
+PROVIDES="
+	hexvexed = $portVersion
+	app:hexvexed = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+	
+ BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:g++
+	"
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects
+	make $jobArgs OBJ_DIR=objects bindcatalogs
+}
+INSTALL()
```

Comment:
I Built it but without dependence the package built but if I build it with dependence  error comes says "libperl.so" is missing.  this error come to a lot of recipes 
 And how to find source files like "cp Soucers\hexvexed  $appDir" here how to find where the app built (how did they found the the app is on Soucers file )